mlflow/server/gc_worker.py

```diff
@@ -0,0 +1,128 @@
+import logging
+import re
+import threading
+import time
+import warnings
+from datetime import timedelta
+
+from mlflow.entities import ViewType
+from mlflow.exceptions import InvalidUrlException, MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.server.handlers import _get_tracking_store
+from mlflow.store.artifact.artifact_repository_registry import get_artifact_repository
+from mlflow.utils.time import get_current_time_millis
+
+_logger = logging.getLogger(__name__)
+
+
+def _gc_once(older_than_ms: int | None = None):
```

Comment:
```suggestion
def _gc_once(older_than_ms: Optional[int] = None):
```

`|` syntax is not supported for Python 3.9.